src/App.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable */
+import './App.css';
+import { Route, Routes, Navigate } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import Reservations from './components/pages/Reservations';
+import Login from './components/auth/Login'
+import Register from './components/auth/Register'
+import Splash from './components/auth/Splash'
+import Nav from './components/left/Nav';
+import Vehicles from './components/pages/Vehicles';
+import VehiclesDetails from './components/pages/VehiclesDetails';
+import AddVehicles from './components/pages/AddVehicles';
+import AddReservation from './components/pages/AddReservation';
+
+const App = () =>{

```

Comment:
- [ ] I still can't find a way to delete vehicles.  As this is a core functionality, if you are not going to implement this requirement, kindly add an explanation in the PR description about what prevented you to complete that step.